### ISSUE TITLE
Start hidden

### ISSUE THREAD

**Issue by lucasmontec** [yasirkula/UnityIngameDebugConsole#2]
Your console is a great debugging tool! It would be great if there was a simple option to start hidden or an alternative prefab to use in this manner.

**Comment by yasirkula** [comment]
Good thinking. Will implement an exposed variable for it shortly.

**Comment by yasirkula** [comment]
Can you check the latest version? There is a new variable called "Launch In Popup Mode" in DebugLogCanvas prefab.

**Comment by lucasmontec** [comment]
I'm getting a Null Reference Exception on the method call for OnSetVisible() because OnEnable for DebugLogManager is called before the awake for the DebugLogPopup. To fix this I moved the entire 'if (launchInPopupMode)' to Start on DebugLogManager class. This solved the issue.

tl;dr canvasGroup is not set when onEnable is called in DebugLogManager, this results in a NRE.

**Comment by yasirkula** [comment]
I did not encounter that error while testing miself, strangely, but I will apply your solution to the main branch soon.

**Comment by lucasmontec** [comment]
Probably an error with unet.I have also changed your LogFilter name since
unet uses an identical name.

Em 11/10/2016 02:21, "Süleyman Yasir KULA" <notifications@github.com>
escreveu:

**Comment by yasirkula** [comment]
I have moved the launchInPopupMode code to Start function and changed the name of LogFilter to DebugLogFilter. Thanks for all the info.

**Comment by lucasmontec** [comment]
Thanks for the awesome asset!

Em 11/10/2016 07:47, "Süleyman Yasir KULA" <notifications@github.com>
escreveu:
